src/EtlLib/Support/InputOutputAdapter.cs

```diff
@@ -102,7 +102,7 @@ public bool AttachConsumer(ISinkNode<T> input)
             if (input.Input != null)
                 throw new InvalidOperationException($"Node (Id={input.Id}, Type={input.GetType().Name}) already has an input assigned.");
 
-            if (!_queueMap.TryAdd(input, new BlockingCollection<T>(new ConcurrentQueue<T>())))
+            if (!_queueMap.TryAdd(input, new BlockingCollection<T>(new ConcurrentQueue<T>(), 10000)))

```

Comment:
What was the purpose of this?  Did you run into a problem?  Should be configurable via EtlLibConfig probably instead.